README.md

```diff
@@ -18,48 +18,123 @@ Answer the following data queries. Keep track of the SQL you write by pasting it
 
 ### find all customers that live in London. Returns 6 records.
 > This can be done with SELECT and WHERE clauses
+```
+SELECT *
+FROM [Customers]
+where city = 'London'
+```
 
 ### find all customers with postal code 1010. Returns 3 customers.
 > This can be done with SELECT and WHERE clauses
 
+```
+SELECT *
+FROM [Customers]
+where PostalCode = '1010'
+```
+
 ### find the phone number for the supplier with the id 11. Should be (010) 9984510.
 > This can be done with SELECT and WHERE clauses
+```
+SELECT Phone
+FROM [Suppliers]
+where SupplierID = '11'
+```
 
 ### list orders descending by the order date. The order with date 1997-02-12 should be at the top.
 > This can be done with SELECT, WHERE, and ORDER BY clauses
+```
+SELECT *
+from [Orders]
+where OrderDate
+order by OrderDate desc
+```
 
 ### find all suppliers who have names longer than 20 characters. You can use `length(SupplierName)` to get the length of the name. Returns 11 records.
 > This can be done with SELECT and WHERE clauses
+```
+SELECT *
+FROM [Suppliers]
+where length(SupplierName) > 20
+```
 
 ### find all customers that include the word "market" in the name. Should return 4 records.
 > This can be done with SELECT and a WHERE clause using the LIKE keyword
-
 > Don't forget the wildcard '%' symbols at the beginning and end of your substring to denote it can appear anywhere in the string in question
 
+```
+SELECT CustomerName
+FROM [Customers]
+where CustomerName like'%market%'
+```
+
 ### add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.
 > This can be done with the INSERT INTO clause
 
+```
+insert
+into Customers(CustomerName, ContactName, Address, City, PostalCode, Country)
+values('the shire', 'bilbo baggins', '1-hobbit hole', 'bag end', '111', 'middle earth')
+```
+
+
 ### update _Bilbo Baggins_ record so that the postal code changes to _"11122"_.
 > This can be done with UPDATE and WHERE clauses
 
+```
+update Customers
+set PostalCode = '11122'
+where CustomerName = 'the shire'
+```
+
+
 ### list orders grouped by customer showing the number of orders per customer. _Rattlesnake Canyon Grocery_ should have 7 orders.
 > This can be done with SELECT, COUNT, JOIN and GROUP BY clauses. Your count should focus on a field in the Orders table, not the Customer table
```

Comment:
Since this request does not explicitly mention returning anything else aside from the order count, you can perform this query without having to use a `JOIN` clause (though this doesn't make your current query incorrect, since it does add clarity to the results of the original request):
```
SELECT CustomerID, COUNT(OrderID) OrderCount
FROM Orders
GROUP BY CustomerID
```